ui.tests/test-module/libs/support/commands.js

```diff
@@ -330,6 +332,22 @@ Cypress.Commands.add("cleanTest", (editPath) => {
     });
 })
 
+Cypress.Commands.add("cleanTitleTest", (editPath) => {
```

Comment:
can we have something reusable for all components if required. something like `removeComponents`
Also ,it's not being used in sites editor test. any reasons? we should be consistent about its usage here  otherwise it may confuse the developers in future whether to include it or not everytime.

Reply:
Hi @nit23uec , we already have a generic command named cleanTest. Will add this in the site test as well